jetbrains-core/tst/software/aws/toolkits/jetbrains/services/telemetry/TelemetryServiceTest.kt

```diff
@@ -188,6 +181,13 @@ class TelemetryServiceTest {
         assertMetricEventsContains(eventCaptor.allValues.flatten(), "Foo", "222222222222", "bar-region")
     }
 
+    private fun createMockBatcher() = mock<TelemetryBatcher> {
+        on { enqueue(any<MetricEvent>()) }.then {
+            mock.enqueue(listOf(it.getArgument<MetricEvent>(0)))
```

Comment:
why are we still enqueing list? didn't you remove that one?

Reply:
That...might explain my test failures haha. I didn't locally run them since it makes my Mac battery die too quickly